test_data/test_route/texts/beol/beolTEI.py

```diff
@@ -12,7 +12,7 @@
         'restype_id': 'http://www.knora.org/ontology/knora-base#XSLTransformation',
```

Comment:
is this file ever used? when and how? (and why?)

Reply:
I haven't found direct usage of it in the repo. @subotic could you bring some light if we still need it? It was added with https://github.com/dasch-swiss/dsp-api/issues/862

Reply:
This seems to be a script that creates a BEOL TEI-XML mapping, which is necessary so that the BEOL data could be exported as TEI-XML. Personally I would leave it. Either we remove everything that we don't need or leave everything as is, as it could maybe useful in the future.